linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/ansible/shared.yml

```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+{{{ ansible_instantiate_variables("rsyslog_remote_loghost_address") }}}
+
+- name: "Get omfwd configuration directive"
+  shell: sed -e '/action\s*(\s*type\s*=\s*"omfwd"/,/)/!d' /etc/rsyslog.conf /etc/rsyslog.d/*.conf
+  register: include_omfwd_config_output
+
+- name: "Get include files directives"
+  shell: "echo \"{{ include_omfwd_config_output.stdout }}\"|grep  'StreamDriver=\"gtls\"'"
```

Comment:
It should probably be:
```
  shell: >
    set -o pipefail
    echo \"{{ include_omfwd_config_output.stdout }}\" | grep 'StreamDriver=\"gtls\"'
```

because there is this in Ansible lint:

```

risky-shell-pipe: Shells that use pipes should set the pipefail option
build/ansible_roles/rhel8_anssi_bp28_intermediary/tasks/main.yml:5832 Task/Handler: Get include files directives

Error: risky-shell-pipe Shells that use pipes should set the pipefail option
```

Reply:
Should be ok in 8e4a9f8